src/app/containers/FrontPageStoryRows/index.jsx

```diff
@@ -1,49 +1,141 @@
 import React from 'react';
 import Grid from '@bbc/psammead-grid';
-import { node, arrayOf, shape, any } from 'prop-types';
+import { arrayOf, shape, bool, oneOf } from 'prop-types';
+import styled, { css } from 'styled-components';
+
+import {
+  GEL_GROUP_1_SCREEN_WIDTH_MIN,
+  GEL_GROUP_2_SCREEN_WIDTH_MAX,
+} from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+import { StoryPromoLi } from '@bbc/psammead-story-promo-list';
```

Comment:
The styling for StoryPromoLi includes a border-bottom that's visible at every breakpoint. 
We'll want to update this so that it is only visible at breakpoints <= 1008px. 

<img width="600" alt="Screenshot of persian front page with the border bottoms visible" src="https://user-images.githubusercontent.com/3028997/73166528-efa1b300-40ed-11ea-991b-808c7e3c5148.png">

Reply:
https://github.com/bbc/psammead/pull/3014